doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -124,6 +124,10 @@ Pelion client
 Thingy:53: Matter weather station
 ---------------------------------
 
+* Added:
+  * Support for Power Source cluster to expose information about the battery.
+  * Support for Identify cluster, using the built-in buzzer to ease finding the device.
```

Comment:
```suggestion
* Added:

  * Support for the Power Source cluster, used to expose information about the battery.
  * Support for the Identify cluster, which uses the built-in buzzer to help find the device.
```
Are these additional clusters visible in the Android CHIPTool mobile app next to Sensor Clusters?

Reply:
No, Android CHIPTool currently doesn't support these clusters.